pylxd/image.py

```diff
@@ -80,7 +81,66 @@ def create(cls, client, image_data, public=False, wait=False):
             Operation.wait_for_operation(client, response.json()['operation'])
         return cls(client, fingerprint=fingerprint)
 
+    @classmethod
+    def create_from_config(cls, client, config, wait=False):
+        """ Create an image from the given configuration.
+
+        See: https://github.com/lxc/lxd/blob/master/doc/rest-api.md#post-6
+        """
+        if 'source' not in config or 'fingerprint' not in config['source']:
+            raise KeyError(
+                'You need to provide a fingerprint '
+                'in the config[\'source\'] argument'
+            )
+
+        response = client.api.images.post(json=config)
+        if wait:
+            Operation.wait_for_operation(client, response.json()['operation'])
+        return cls(client, fingerprint=config['source']['fingerprint'])
+
     def export(self):
         """Export the image."""
         response = self.api.export.get()
         return response.content
+
+    def copy_to_lxd(self, new_client, public=None, auto_update=None,
+                    wait=False):
+        """ Copy an image to a another LXD.
+
+        Destination host information is contained in the client
+        connection passed in.
+        """
+        self.sync()  # Make sure the object isn't stale
+
+        # TODO: this is a hacky way to remove the /1.0 do we have a better one?
```

Comment:
Oh, I forgot to point out that we don't use `TODO` for acknowledging technical debt. You should use this format:

    # XXX: pcdummy (28 Jul 2016) - This is a hacky way....

The formatting is important (including the date), as it is consistent with the rest of the project and many tools.